WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -1546,43 +1546,31 @@ local function modify(parent, region, data)
         region:RealClearAllPoints()
         region:SetPoint("CENTER", UIParent, "CENTER")
       end
-      for active, regionData in ipairs(self.sortedChildren) do
+      for _, regionData in ipairs(self.sortedChildren) do
         if regionData.shown then
-          numVisible = numVisible + 1
-          local childRegion = regionData.region
-          local regionLeft, regionRight, regionTop, regionBottom
-             = SafeGetPos(childRegion, childRegion.GetLeft), SafeGetPos(childRegion, childRegion.GetRight),
-               SafeGetPos(childRegion, childRegion.GetTop), SafeGetPos(childRegion, childRegion.GetBottom)
-
-          if(regionLeft and regionRight and regionTop and regionBottom) then
-            minX = minX and min(regionLeft, minX) or regionLeft
-            maxX = maxX and max(regionRight, maxX) or regionRight
-            minY = minY and min(regionBottom, minY) or regionBottom
-            maxY = maxY and max(regionTop, maxY) or regionTop
-          end
+          anyVisible = true
+          break
         end
       end
 
-      if numVisible > 0 then
+      if anyVisible then
         self:Show()
-        minX, maxX, minY, maxY = (minX or 0), (maxX or 0), (minY or 0), (maxY or 0)
-
-        local width, height = maxX - minX, maxY - minY
+        self:SetSize(1, 1)
+        local left, bottom, width, height = self:GetBoundsRect()
```

Comment:
Well.... Sorf of, there's a philosophical question here, what's the bounded rect of an icon aura with a text that that is below it. GetBoundsRect() returns something different thatn the previous code.

Now, the size calculation here is imho wrong anyway, since it takes the current actual region position at the time of update into account. If the aura is animating at the point, it uses the region's dynamic position at that moment, but doesn't update later as the animation continues.

That has lead to glorious fixes like https://github.com/InfusOnWoW/WeakAuras2/commit/206e179447b636ba4e42e3d385a328633df6d7f2, and the follow up https://github.com/InfusOnWoW/WeakAuras2/commit/3ae2a2f50cbc77073e3b3eec9ef81201b69e5dc1

Using GetBoundsRect() also has that problem, that we don't know when it changes, and thus don't reliable can update it.

So I'd rather not touch that code, unless we refactor it entirely and make it have clean semantics and robust.